README.md

```diff
@@ -234,8 +230,7 @@ explicitly opt for building in simulation-mode SGX, ``SGX_MODE=SIM``. In order t
 location, or for building in simulation-mode SGX, you can create the file `config.override.mk` and override the default
 values by defining the corresponding environment variable.
 
-
-##### Intel Attestation Service (IAS)
+#### Intel Attestation Service (IAS)
```

Comment:
splitting the section in the two options docker vs native install is a good idea. However, as the IAS setup applies to both settings you have to move this to a different place, probably best just before the two options (and with one '#' less)

Reply:
nope, options 1 and two are on level ### and SDK & SSL are both at level #### under option 2. Also why should IAS be a subsetion of SDK & SSL?  Having SDK & SSL under option 2 makes sense as for option 1 it is pre-installed. But IAS credentials you need in both cases

Reply:
Moved SGX and IAS sections to top.